tests/functional/globals.py

```diff
@@ -1,13 +1,27 @@
 import os
 
+def get_builtin_modules():
+    module_path = os.popen("%s --version | grep \"Module Path\" | cut -d ' ' -f 3" % get_syslog_ng_binary(), 'r').read().split(".")
+    return '%s:' % module_path
```

Comment:
We should probably use a dash in the name of "Module-Path", or rather Module-Directory and also a colon to keep the rfc822 style formatting of --version output.